sagemaker_model_monitor/fairness_and_explainability_jsonlines/SageMaker-Monitoring-Feature-Attribution-Drift-for-Endpoint.ipynb

```diff
@@ -0,0 +1,1561 @@
+{
```

Comment:
<p>I did not understand meaning of <code>without being encoding by BASE64</code> here.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>To avoid confusion I will change the sentence to</p><p> </p><p>* Payloads are typically treated as binary data and encoded in BASE64 by default, allowing them to be stored in capture data files. However, by specifying the data format in the <code>json_content_types</code> parameter as shown below, the payloads can be captured as plain text instead.</p>